recipe/build.sh

```diff
@@ -109,4 +118,21 @@ echo ">>>>>>>"
 echo "Showing version-info.inc: "
 cat obj_cmake/Src/version-info.inc
 echo ">>>>>>>"
-cmake --build obj_cmake --target install
+cmake --build obj_cmake -j 2 --target install
+
+
+# Run tests in the build-directory, this is important since the tests folders
+# get deleted after build!
+
+# This is just to ensure it works *better* on lone machines.
+# Users on clusters should do something differently,
+# or unset these.
+export OMPI_MCA_plm=isolated
+export OMPI_MCA_btl_vader_single_copy_mechanism=none
+export OMPI_MCA_rmaps_base_oversubscribe=yes
+
+
+echo "Running tests"
+pushd obj_cmake/Tests/08.GeometryOptimization
+SIESTA_TESTS_VERIFY=1 ctest -L simple || echo "Accepted fail!"
+popd
```

Comment:
Is there a specific reason to move the test from `run_test.sh` to `build.sh`?

Reply:
because conda cleans source dirs, and I need the cmake build dir... :(